src/scripts/evaluate_llm_benchmark.py

```diff
@@ -0,0 +1,71 @@
+"""Script to evaluate LLM benchmark."""
+
+import logging
+
+import hydra
+import pandas as pd
+from omegaconf import DictConfig
+
+from european_values.data_loading import load_evs_wvs_data
+from european_values.data_processing import process_data
+from european_values.llm_evaluation import evaluate_survey_data
+
+logger = logging.getLogger("evaluate_llm")
+
+
+@hydra.main(config_path="../../config", config_name="config", version_base=None)
+def main(config: DictConfig) -> None:
+    """Main evaluation function."""
+    # Load data
+    logger.info("Loading data...")
+    df = load_evs_wvs_data()
```

Comment:
Can you replace this with the data loading block from the other scripts? That allows it to be used with the trends dataset as well.